app/src/main/res/layout/activity_offer.xml

```diff
@@ -4,26 +4,234 @@
+                                android:layout_marginStart="@dimen/base_margin_half"
+                                android:orientation="vertical">
+
+                                <TextView
+                                    android:id="@+id/premiumCurrency"
+                                    android:layout_width="wrap_content"
+                                    android:layout_height="wrap_content"
+                                    android:text="@string/OFFER_PREMIUM_CURRENCY_ABBREVIATION"
+                                    android:textAppearance="?textAppearanceHeadline6"
+                                    android:textColor="@color/gray" />
+
+                                <TextView
+                                    android:id="@+id/premiumPeriod"
+                                    android:layout_width="wrap_content"
+                                    android:layout_height="wrap_content"
+                                    android:text="@string/OFFER_PREMIUM_PERIOD_ABBREVIATION"
+                                    android:textAppearance="?textAppearanceHeadline6"
+                                    android:textColor="@color/gray" />
+
+                            </LinearLayout>
+
+                        </LinearLayout>
+
+                        <LinearLayout
+                            android:id="@+id/startDateContainer"
+                            android:layout_width="match_parent"
+                            android:layout_height="wrap_content"
+                            android:layout_margin="@dimen/base_margin_double"
+                            android:background="@drawable/background_pick_date"
+                            android:drawableEnd="@drawable/ic_expand"
+                            android:gravity="center"
+                            android:orientation="horizontal"
+                            android:padding="@dimen/base_margin">
+
+                            <TextView
+                                android:layout_width="0dp"
+                                android:layout_height="wrap_content"
+                                android:layout_weight="1"
+                                android:text="@string/START_DATE_BTN" />
+
+                            <TextView
+                                android:id="@+id/premiumBoxStartDate"
+                                android:layout_width="wrap_content"
+                                android:layout_height="wrap_content"
+                                android:layout_marginEnd="@dimen/base_margin_double"
+                                tools:text="Idag" />
+
+                            <ImageView
+                                android:layout_width="wrap_content"
+                                android:layout_height="wrap_content"
+                                android:src="@drawable/ic_expand" />
+
+                        </LinearLayout>
+
+                        <com.google.android.material.button.MaterialButton
```

Comment:
```suggestion
                        <Button
```

All buttons automatically become `MaterialButton`s when you have a `MaterialComponents`-theme